app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/MyGamesBundleView.kt

```diff
@@ -125,9 +129,13 @@ fun MyGamesBundleViewContent(
           key = { _, it -> it.packageName }
         ) { _, it ->
           val appIcon = rememberAppIconDrawable(packageName = it.packageName, localContext)
+          val (appUiState, _) = rememberApp(packageName = it.packageName, useStoreName = false)
```

Comment:
MyGames apps should not have rating since it can contain apps that are not from AG